src/main/java/hudson/plugins/git/browser/GitBlitRepositoryBrowser.java

```diff
@@ -72,6 +73,7 @@ private String encodeString(final String s) throws UnsupportedEncodingException
     }
 
     @Extension
+    @Symbol("gitBlit")
```

Comment:
I expect the [gitblit-plugin](https://github.com/jenkinsci/gitblit-plugin) would prefer that we leave this symbol for them to define, rather us defining it for a very narrow case of the repository browser in the git plugin.
```suggestion
    @Symbol("gitBlitRepositoryBrowser")
```